src/Illuminate/Foundation/Providers/ArtisanServiceProvider.php

```diff
@@ -226,6 +227,7 @@ class ArtisanServiceProvider extends ServiceProvider implements DeferrableProvid
         'TraitMake' => TraitMakeCommand::class,
         'VendorPublish' => VendorPublishCommand::class,
         'ViewMake' => ViewMakeCommand::class,
+        'ConfigMake' => ConfigMakeCommand::class,
```

Comment:
All other commands seem to be listed in alphabetical order. 

I guess it is best to add this one on the corresponding spot to keep that order.

Reply:
Thanks!